### PR TITLE
ENGT-9213: graphite-web startup failure

### PR DESCRIPTION
**Pull request by earlruby**
Sometimes graphite-web starts, but runit fails to understand that it started,
so timeout the startup after 10 seconds but try to start it at least 10 times.

Here is my change from July 18 that fixed the problem: https://github.com/apcera/continuum-chef/pull/988/files

Here is an update of the recipe from Tom Cahill that reintroduced the original code with the original problem: https://github.com/apcera/continuum-chef/commit/4bf8805e33c54e3ba474033fa0fe000583cf59fa

So basically my fix got overwritten because I added the change to `continuum-chef` and I should have added it to `chef-graphite`.

### CODE REVIEW

File: recipes/uwsgi_runit.rb [hunk 1]
```diff
@@ -1,9 +1,11 @@
 include_recipe 'runit'
 
 runit_service 'graphite-web' do
-  # uwsgi/graphite-web can sometimes be a tad slow to startup, so allow more
-  # than the default 7 seconds for it load up.
-  sv_timeout 120
+  # Sometimes graphite-web starts, but runit fails to understand that it started,
+  # so timeout the startup after 10 seconds but try to start it at least 10 times.
+  sv_timeout 10
```
Comment (davidweishi) [inline recipes/uwsgi_runit.rb]: The comment suggests that graphite-web is slow to startup.  I might be safer to increase sv_timeout to 15 or 20.

Comment (earlruby) [inline recipes/uwsgi_runit.rb]: If you check the logs (referenced in the JIRA), when it "times out" it actually starts in less than 2 seconds, it's just that runit doesn't detect that it started. Looks like cruft from a previous restart remains, and that messes up runit. Restarting a second time clears the cruft and allows runit to detect a proper start.

The change to 120s was made by Dennis here: https://github.com/apcera/chef-graphite/commit/b0041a77136e19885623cb3e195a4121e652b540, but because the problem is intermittent he wasn't sure that that would fix the problem, and in fact the longer timeout didn't actually fix the issue and was unnecessary. Prior to that Ken and Josh had both bumped up the timeout.

If you look at the Graphite log from the [previous ticket where Dennis changed this to 120s](https://apcera.atlassian.net/browse/ENGT-6188), it shows Graphite starting up in 5 seconds.
